src/main/java/com/parasoft/findings/jenkins/parser/FlowAnalysisPathBuilder.java

```diff
@@ -143,21 +128,6 @@ private String getMessage(IFlowAnalysisPathElement descriptor, boolean bFullDesc
 
         if (useAnnotation) {
             addAnnotations(sb, descriptor, bFullDescription);
-        } else {
```

Comment:
Why do we delete this block but keep the similar logic at line 339?
Please give examples to explain the reasons.